crowbar_framework/lib/execute.rb

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2013-2016, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "open3"
+
+class Execute
+  class << self
+    def bash(*params)
```

Comment:
Let's call it `bash_as_user`, than it's more obvious that it's not executed with higher privileges :)